webapp/channels/src/components/quick_switch_modal/quick_switch_modal.tsx

```diff
@@ -48,25 +45,25 @@ export type Props = {
         switchToChannel: (channel: Channel) => Promise<ActionResult>;
         closeRightHandSide: () => void;
     };
-}
+    focusOriginElement: React.RefObject<HTMLElement>;
```

Comment:
this modal only needs the `focusOriginElement` since the component `SuggestionBox` already has set logic to place the focus in the input text box. The `focusOriginElement` will be passed from the logic that triggers the opening of the modal, so the modal will pass this to the `GenericModal` which will use it to restore the focus to that element when the modal hides.